### DISCUSSION TITLE
Nova-2-ea Hi

### DISCUSSION THREAD

**Discussion by alexbainbridge** [deepgram/community#360] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

With 
`model=nova-2-ea&interim_results=true&filler_words=false&smart_format=true`

I have a problem that there are quite a few "Hi" inserted at start of response when the person talking has said absolutely nothing.  It is always at the start of a sentence. 

```json 
{
    "type": "Results",
    "channel_index": [
        0,
        1
    ],
    "duration": 2.33,
    "start": 0,
    "is_final": true,
    "speech_final": true,
    "channel": {
        "alternatives": [
            {
                "transcript": "Hi.",
                "confidence": 0.801134,
                "words": [
                    {
                        "word": "hi",
                        "start": 2,
                        "end": 2.33,
                        "confidence": 0.801134,
                        "punctuated_word": "Hi."
                    }
                ]
            }
        ]
    },
    "metadata": {
        "request_id": "385ce751-ddeb-49ef-9568-62b0749718be",
        "model_info": {
            "name": "2-ea-nova",
            "version": "2023-09-18.22988",
            "arch": "nova-2"
        },
        "model_uuid": "79dd3ac5-776e-4fcf-b4fb-0f9968bac445"
    }
}
```
It is regular. 

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

wss://api.deepgram.com/v1/listen?language=en&model=nova-2-ea&interim_results=true&filler_words=false&smart_format=true

**Comment by rudder-aniket-kale** [comment]
Facing a similar issue on nova-2-ea. Also never faced this issue on nova-general model before.

**Comment by alexbainbridge** [comment]
Hi.  ;)

**Comment by jpvajda** [comment, marked as answer]
@alexbainbridge  @aniket-rudder  I'll share this with one of our Product Teams to evaluate this issue. Thanks for letting us know.
